lib/src/view/play/ongoing_games_screen.dart

```diff
@@ -87,6 +89,12 @@ class OngoingGamePreview extends ConsumerWidget {
                 allowFromNow: true,
               ),
             ),
+          if (game.secondsLeft! > 0)
```

Comment:
You cannot use the non-null assertion operator without checking null first.

Reply:
added